arch/arm/src/qemu/qemu_boot.c

```diff
@@ -36,6 +36,11 @@
 #  include <nuttx/fdt.h>
 #endif
 
+#ifdef CONFIG_SCHED_INSTRUMENTATION
+#  include <sched/sched.h>
+#  include <nuttx/sched_note.h>
```

Comment:
```suggestion
#  include <nuttx/sched_note.h>
```

Reply:
The definition of this_task is included in <sched/sched.h>
nuttx/sched_note.h does not include <sched/sched.h>

Reply:
ok